httpd/handler.go

```diff
@@ -148,7 +116,7 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 
 	// If authentication is enabled and there are no users yet, make sure
 	// the first statement is creating a new cluster admin.
-	if h.AuthenticationEnabled && h.server.UserCount() == 0 {
+	if h.requireAuthentication && h.server.UserCount() == 0 {
```

Comment:
Enabled typically means the feature is turned on.  It doesn't convey that we actually need you to be authorized now.

Required has a completely different meaning so I purposefully changed the word for clarity.